docs/ui/components/Sidebar/SidebarCollapsible.tsx

```diff
@@ -63,7 +63,8 @@ export function SidebarCollapsible(props: Props) {
       <a css={titleStyle} onClick={toggleIsOpen}>
         <div css={chevronContainerStyle}>
           <ChevronDownIcon
-            size={iconSize.small}
+            size={iconSize.tiny}
+            color={theme.icon.secondary}
```

Comment:
```suggestion
```
I still think these are better as the default icon color. I think our secondary icon color is too light. It might be that we need a new color for secondary, and that we make the current secondary color a new "tertiary" color.

For now though, I think we should use the regular icon color.